proto/internalapi/scanner/v4/index_report.proto

```diff
@@ -0,0 +1,23 @@
+// This contains protobuf types in pair with ClairCore's types. See
+// https://github.com/quay/claircore for comments on the fields.
+
+syntax = "proto3";
+
+package v4;
+
+import "google/protobuf/timestamp.proto";
+import "internalapi/scanner/v4/common.proto";
+
+option go_package = "v4";
+
+message IndexReport {
+    string hash_id    = 1;
+    string state   = 2;
+    bool   success = 3;
+    string err     = 4;
+
+    repeated Package packages                  = 5;
+    repeated Distribution distributions        = 6;
+    repeated Repository repositories           = 7;
+    map<string, Environment.List> environments = 8;
```

Comment:
I can't make the suggestion directly on GitHub because it's annoying with the spacing, but can you align the variable names?

Reply:
8a31bbb545..0eb2ddb78d